src/components/ChatLog.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+  const chats = props.entries.map((entry, i) => {
+    return (
+      <ChatEntry
+        key={i}
+        id={entry.id}
+        sender={entry.sender}
+        body={entry.body}
+        timeStamp={entry.timeStamp}
+        liked={entry.liked}
+        updateLike={props.updateLike}
+      />
+    );
+  });
+  return (
+    <div>
+      <div>{chats}</div>
+    </div>
+  );
+};
+
```

Comment:
Remember to put your props-types checking for this one. You could use the `arrayOf` and then `shape` functionality to be very explicit about the data structure you are expecting to pass through here, `entries`.